content/ko/docs/reference/glossary/kubeadm.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Kubeadm
```

Comment:
```suggestion
title: kubeadm
```

Reply:
Hi @sftim  :)

Do you think this needs to be applied to upstream (English) ?
Currently, the title of content/en/docs/reference/glossary/kubeadm.md is **K**ubeadm

https://raw.githubusercontent.com/kubernetes/website/main/content/en/docs/reference/glossary/kubeadm.md